src/new.rs

```diff
@@ -521,7 +521,7 @@ fn validate_name(name: String) -> Result<String, Error> {
             name,
             reason: InvalidProjectNameReason::ErlangStandardLibraryModule,
         })
-    } else if crate::parser::is_gleam_reserved_word(name.as_str()) {
+    } else if crate::parse::lexer::str_to_keyword(name.as_str()).is_some() {
```

Comment:
Don't gotta maintain a list of reserved words anymore, the lexer has to know about them. so much for all those PRs lol.

Reply:
Hey this is really nice, we won't make that mistake again